back/app/controllers/web_api/v1/users_controller.rb

```diff
@@ -13,7 +13,7 @@ def index
 
     @users = @users.search_by_all(params[:search]) if params[:search].present?
 
-    @users = @users.active unless params[:include_inactive]
+    @users = @users.active_and_blocked unless params[:include_inactive]
```

Comment:
This may look weird, but it is intended.

Our back office `/admin/users` 'Registered Users` view, in fact, includes active users & blocked users. This seems to make sense, as it gives a central list that can be searched for blocked and unblocked users, but that also excludes inactive users (not completed registration).

I have attempted to keep all affected/new scopes logical and explicit in their naming.

Reply:
Now using `:registered` name for this scope.